packages/editor-ui/src/router.ts

```diff
@@ -120,7 +120,7 @@ export const routes = [
 				getProperties(route: RouteLocation) {
 					const templatesStore = useTemplatesStore();
 					return {
-						template_id: route.params.id,
+						template_id: isNaN(+route.params.id) ? route.params.id : +route.params.id,
```

Comment:
Maybe this could be its own function (e.g. `ensureTemplateIdNumber` or something) since it's used in many places now.